AnalyticsService/app/routes.py

```diff
@@ -48,3 +48,13 @@ def create_budget():
     db.session.commit()
 
     return jsonify({"message": "Budget successfully created"}), 201
+
+@routes.route("/analytics/budget", methods=['GET'])
```

Comment:
This route should only be accessible to people with a valid JWT, I would add that check. Also, if the client is directly calling this route, the client will temporarily hold budget data from other companies which is a security risk. I think it will be better for only the budget data for the specific company to get retrieved

Ex: User is Logged into App -> Requests budget data with JWT  -> Route extracts JWT and the company_id parameter -> Return specific company budget data to client

Reply:
Hey Stuart, thanks for addressing this! What you mentioned is something I plan on doing in the future. For now I just wanted to make sure I could retrieve the data properly and render it to the client.